docs/source/cpp/compute.rst

```diff
@@ -171,8 +171,8 @@ To avoid exhaustively listing supported types, the tables below use a number
 of general type categories:
 
 * "Numeric": Integer types (Int8, etc.) and Floating-point types (Float32,
-  Float64, sometimes Float16).  Some functions also accept Decimal128 and
-  Decimal256 input.
+  Float64, sometimes Float16).  Some functions also accept
+  Decimal128/256 input.
```

Comment:
More and more functions now also accept decimal32/decimal64.
```suggestion
  Float64, sometimes Float16).  Some functions also accept Decimal input.
```

Reply:
The initial change to this line was to say that "some functions also accept Decimal32/64/128/256 input" but that change was rolled back to make this PR focus on Decimal128/256. Hence the change here. But this seems fine.